Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.py

```diff
@@ -1287,53 +1297,335 @@ def add_note_to_asset_command(client: Client, args: dict[str, Any]) -> CommandRe
     Returns:
         CommandResults: A CommandResults demisto object that is then passed to return_results
     """
-    asset_id = str(args.get("asset_id"))
-    entity_type = str(args.get("entity_type"))
-    note_to_add = str(args.get("note_to_add"))
-    should_append = argToBoolean(args.get("should_append"))
-
-    response = client.add_note_to_asset(
-        asm_asset_id=asset_id, entity_type=entity_type, annotation_note=note_to_add, should_append=should_append
-    )
-    response_message = {"status": response.get("reply", {})}
-    response_message["asset"] = asset_id
-    markdown = tableToMarkdown(
-        "Add Note to Asset Command Results:", response_message.get("status"), headers=["Status"], removeNull=True
-    )
+    asset_id = str(args.get('asset_id'))
+    entity_type = str(args.get('entity_type'))
+    note_to_add = str(args.get('note_to_add'))
+    should_append = argToBoolean(args.get('should_append'))
+
+    response = client.add_note_to_asset(asm_asset_id=asset_id,
+                                        entity_type=entity_type,
+                                        annotation_note=note_to_add,
+                                        should_append=should_append)
+    response_message = {"status": response.get('reply', {})}
+    response_message['asset'] = asset_id
+    markdown = tableToMarkdown('Add Note to Asset Command Results:',
+                               response_message.get('status'),
+                               headers=['Status'],
+                               removeNull=True)
     command_results = CommandResults(
-        outputs_prefix="ASM.AssetAnnotation",
-        outputs_key_field="",
+        outputs_prefix='ASM.AssetAnnotation',
+        outputs_key_field='',
         outputs=response_message,
         raw_response=response,
-        readable_output=markdown,
+        readable_output=markdown
     )
 
     return command_results
 
 
-def ip_command(client: Client, args: dict[str, Any]) -> CommandResults:
-    command_results = CommandResults(
-        outputs_prefix="",
-        outputs_key_field=None,
-        outputs=None,
-        raw_response=None,
-        readable_output="The `!ip` command for the Cortex Xpanse integration has been deprecated, "
-        "please use the `asm-get-asset-internet-exposure` command or the "
-        "[Xpanse Feed Integration](https://xsoar.pan.dev/docs/reference/integrations/xpanse-feed).",
-    )
+def ip_command(client: Client, args: dict[str, Any]) -> list[CommandResults]:
```

Comment:
This change makes it seem like you weren't branching from master.  Can you please make sure you merge from master?